### ISSUE TITLE
Rust spikes as a source of iron?

### ISSUE THREAD

**Issue by Angular-Angel** [TerraformersMC/Cinderscapes#63]
For nether survival, the nether could use a source of iron. Besides nether fortresses. Maybe some kind of horrible rust spikes, that damage you when you get too close, and maybe also collapse under the wrong circumstances?

**Comment by Angular-Angel** [comment]
Also, pools of liquid gold sound really cool, though they should probably be rare and guarded by very dangerous enemies - Tons of blazes, at the very least. And on the subject of enemies, the luminescent grove and quartz canyon are both too well lit - I would suggest either giving them less light, or making them more dangerous. The quartz canyon in particular could use some more danger to counterbalance the abundance of riches all that quartz represents. Perhaps the quartz should cut you if you try and sprint while touching it, and/or a new enemy?

**Comment by Angular-Angel** [comment]
And, if you're just looking for more stuff to add, another source of blackstone that could be found in any biome would be cool. But it should have some kind of danger associated with it, as well.

**Comment by saltyseadoggo** [comment]
For the record, vanilla Minecraft already has obtainable iron in the Nether. It can be found non-renewably in Nether fortresses and bastion remnants in ingot and nugget formes, and renewably by bartering with Piglins for iron nuggets. I think these rust spikes could have their own merit as unique terrain features, but their niche as an iron source is already filled.

**Comment by Angular-Angel** [comment]
Yeah, but that doesn't explain where the piglins get it. They can't have gotten it all from raiding nether fortresses. And having another source would be really cool, as piglins are super unreliable about actually giving you what you want. XD
